frontend/src/Components/Conditions/ConditionsMap.tsx

```diff
@@ -405,120 +377,51 @@ const ConditionsMap = (props: any) => {
 
   return (
     <div style={{ height: '100%' }}>
-      <div style={{ height: '85%' }}>
-        <MapContainer
-          preferCanvas={true}
-          center={center}
-          zoom={zoom}
-          minZoom={minZoom}
-          maxZoom={maxZoom}
-          scrollWheelZoom={true}
-          zoomControl={false}
-        >
-          <TileLayer
-            maxNativeZoom={maxZoom}
-            maxZoom={maxZoom}
-            attribution='&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
-            url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
+      <div className="nav-wrapper">
+        <div className="nav-container">
+          <Search
+            onPlaceSelect={(value: any) => {
+              console.log(value);
+            }}
           />
-
+        </div>
+        <div className="filter-container">
+          <select
+            className="sweetalert-input"
+            defaultValue={mode}
+            onChange={inputChange}
+            style={{ width: '100px' }}
+          >
+            {conditionTypes.map((value) => (
+              <option value={value} key={value}>
+                {value}
+              </option>
+            ))}
+          </select>
+        </div>
+        <div className="nav-container">
+          <MonthFilter
+            onStartChange={(date: any) => {
+              rangeChange(dateChange(date), true);
+            }}
+            onEndChange={(date: any) => {
+              rangeChange(dateChange(date), false);
+            }}
+          />
+        </div>
+      </div>
+      <div style={{ height: '85%' }}>
```

Comment:
```suggestion
      <div style={{ height: '100%' }}>
```
So that the map take the all page